sites/www.antislaverymanuscripts.org.conf

```diff
@@ -9,7 +9,7 @@ server {
     }
 
     location / {
-        rewrite (?i)\.(jp(e)?g|gif|png|ico|ico|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|ttf|tar|gz|tgz|bz2|tbz2|zip)$ https://static.zooniverse.org/www.antislaverymanuscripts.org$uri;
+        rewrite (?i)\.(jp(e)?g|gif|png|ico|woff|ttf|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|ttf|tar|gz|tgz|bz2|tbz2|zip)$ https://static.zooniverse.org/www.antislaverymanuscripts.org$uri;
```

Comment:
Good catch on the duplicate `ico`.

`ttf` is already in the list,  so this just needs the addition of `woff`.

Reply:
Dangit, did I remove a duplicate only to introduce another duplicate??

Reply:
Yup :)